pulsaradmin/pkg/admin/schema.go

```diff
@@ -130,6 +133,25 @@ func (s *schemas) GetSchemaInfoByVersion(topic string, version int64) (*utils.Sc
 	return info, nil
 }
 
+func (s *schemas) GetAllSchemas(topic string) ([]*utils.SchemaInfoWithVersion, error) {
+	topicName, err := utils.GetTopicName(topic)
+	if err != nil {
+		return nil, err
+	}
+	var response utils.GetAllSchemasResponse
+	endpoint := s.pulsar.endpoint(s.basePath, topicName.GetTenant(), topicName.GetNamespace(),
+		topicName.GetLocalName(), "schemas")
+
+	err = s.pulsar.Client.Get(endpoint, &response)
+	if err != nil {
+		fmt.Println("err:", err.Error())
```

Comment:
I don't think you should print the error here.

Reply:
Removed. It came together with copy pasting `GetSchemaInfoWithVersion`.